README.md

```diff
@@ -1,83 +1,101 @@
-* If you have a partial solution, **still check in a partial solution**
-* You must submit a pull request to this repo with your code by 9am Monday morning
-
-Task
------
 
-* Fork this repo
-* Run the command 'bundle' in the project directory to ensure you have all the gems
-* Write a Takeaway program with the following user stories:
+User Stories
+-----
 
 ```
+User story 1:
 As a customer
 So that I can check if I want to order something
 I would like to see a list of dishes with prices
 
+User story 2:
 As a customer
 So that I can order the meal I want
 I would like to be able to select some number of several available dishes
 
+User story 3:
 As a customer
 So that I can verify that my order is correct
 I would like to check that the total I have been given matches the sum of the various dishes in my order
 
+User story 4:
 As a customer
 So that I am reassured that my order will be delivered on time
 I would like to receive a text such as "Thank you! Your order was placed and will be delivered before 18:52" after I have ordered
 ```
 
-* Hints on functionality to implement:
-  * Ensure you have a list of dishes with prices
-  * The text should state that the order was placed successfully and that it will be delivered 1 hour from now, e.g. "Thank you! Your order was placed and will be delivered before 18:52".
-  * The text sending functionality should be implemented using Twilio API. You'll need to register for it. It’s free.
-  * Use the twilio-ruby gem to access the API
-  * Use the Gemfile to manage your gems
-  * Make sure that your Takeaway is thoroughly tested and that you use mocks and/or stubs, as necessary to not to send texts when your tests are run
-  * However, if your Takeaway is loaded into IRB and the order is placed, the text should actually be sent
-  * Note that you can only send texts in the same country as you have your account. I.e. if you have a UK account you can only send to UK numbers.
+My approach
+-----
+
+User story 1:
+
+- created a menu class. It is responsible for creating the menu. On initialization it is given the attribute @dishes which is equal to an array hashes. Each hash represents a dish, with a key/value pair of name/price. The menu class is also responsible for showing the menu via the view_menu method. It outputs a readable list showing each dish name and it's price.
+
+User story 2:
+- created an order class. It is responsible for creating a basket, adding dishes to the basket and displaying the basket. On initialization it is given the attribute @basket which is an empty array. Adding a dishes adds the dishes to @basket. Displaying the basket shows the dishes in the basket and their prices.
+
+User story 3:
+- added responsibility for calculating and displaying the cost of the basket to the order class. The total cost method shows the sum of the price for each dish in the basket.
+
+User story 4:
+- I ran out of time
+
+
```

Comment:
This section clearly explains what you did and why.  Could you perhaps explain how you'd go about completing US4 if you had more time to complete it?